### DISCUSSION TITLE
Errors Creating collection offer via sdk

### DISCUSSION THREAD

**Discussion by wainesiro** [ProjectOpenSea/opensea-js#1516] [Q&A]
Hello i have been using Sdk to make collection offers and it works perfectly well for some slugs and i keep getting different errors for some and it automatically gets fixed and bids on retry. the error looks like "Error: Server Error: You have attempted to create a order, but you provided an offer item or consideration item with a 0 start amount. Remove this 0-amount item if this was intended."I get the following error on som slugs and it keeps on changing for example right now its "the-crypt-official", "xtremeverse404" etc etc and i also get a error  "ITS NOT A VALID JSON"

**Comment by aUsABuisnessman** [comment]
I would like to know about this too! also my art is showing that i use bitcoin but i should be using Eth?
